sweep/tx_input_set.go

```diff
@@ -393,6 +394,12 @@ func (t *txInputSet) tryAddWalletInputsIfNeeded() error {
 		return err
 	}
 
+	// Sort the UTXOs by putting smaller values at the start of the slice
+	// to avoid locking large UTXO for sweeping.
+	sort.Slice(utxos, func(i, j int) bool {
```

Comment:
can we do this in the wallet lib maybe or what be really slick connect it to the lnd parameteer where we select random or ordered wallet inputs: https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf#L149

Reply:
oh yeah forgot about that config! added a todo here and we can fix it in later refactoring PRs.

Reply:
`btcwallet` does the sorting rn when it does its own coin selection for `SendCoins`. The existing behavior for `ListUnspent` is to return in decreasing order IIUC. We can thread through a new arg to allow that behavior to be modified, but don't think it's a blocking factor here.